gui/layers/_markers_layer/view/controls.py

```diff
@@ -0,0 +1,147 @@
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QIcon
+from PyQt5.QtWidgets import QButtonGroup, QVBoxLayout, QRadioButton, QFrame
+
+from os.path import join
+from ....resources import resources_dir
+
+
+class QtMarkersControls(QFrame):
+    def __init__(self, layer):
+        super().__init__()
+
+        self.layer = layer
+        self.select_button = QtSelectButton(layer)
+        self.addition_button = QtAdditionButton(layer)
+        self.panzoom_button = QtPanZoomButton(layer)
+
+        self.button_group = QButtonGroup(self)
+        self.button_group.addButton(self.select_button)
+        self.button_group.addButton(self.addition_button)
+        self.button_group.addButton(self.panzoom_button)
+
+        layout = QVBoxLayout()
+        layout.addWidget(self.select_button)
+        layout.addWidget(self.addition_button)
+        layout.addWidget(self.panzoom_button)
+        layout.addStretch(0)
+        self.setLayout(layout)
+        self.setMouseTracking(True)
+
+    def mouseMoveEvent(self, event):
+        self.layer.viewer.status = self.layer.status
+
+    # def on_key_press(self, event):
```

Comment:
What was all of this for?

Reply:
This was how we did all the key press stuff before, like entering annotation mode or space bar to get back into zoom mode. I will reimplement, can either do in this PR or the next one.